rest-api/src/main/resources/paths/index.yml

```diff
@@ -276,6 +276,8 @@
     $ref: ./participants/v3_participants_healthCode.yml
 /v3/participants/syn:{synapseUserId}:
     $ref: ./participants/v3_participants_synapseUserId.yml
+/v3/participants/{userId}/enrollments:
+    $ref: ./participants/v3_participants_userId_enrollments.yml
```

Comment:
This was missing from the SDK despite being present on the server...I discovered this because I needed to change the paging type returned to be a paginated list.